docs/user/guides/set-up-single-sign-on-saml.rst

```diff
@@ -0,0 +1,125 @@
+
+7. Click :guilabel:`Next`.
+8. Select ``This is an internal app that we have created``.
+9. Click :guilabel:`Finish`.
+
+User setup
+~~~~~~~~~~
+
+Using this setup, all users who have access to the configured Okta application will automatically join to your organization when they sign up.
+Existing users will not be automatically joined to the organization.
+
+You can still add outside collaborators and manage their access.
+There are two ways to manage this access:
+
+* Using :doc:`teams </guides/manage-read-the-docs-teams>` to provide access for ongoing contribution.
+* Using :doc:`sharing </commercial/sharing>` to provide short-term access requiring a login.
+
+Enabling SSO
+------------
+
+Enabling SSO is currently done by the Read the Docs team,
+contact :doc:`support </support>` to enable this feature for your organization.
+
+By default, users that sign up with SAML do not have any permissions over any project.
+However, you can define which teams users will auto-join when they sign up.
+
+After enabling the SAML integration,
+all users with email addresses from your configured domain will be required to signup using SAML.
+
+.. warning::
+
+   Existing users with email addresses from your configured domain will not be required to sign up using SAML,
+   but they won't be automatically joined to your organization.
+
+Configure team for all users to join
+------------------------------------
+
+You can mark one or many teams that users are automatically joined when they sign up with a matching email address.
+Configure this option by:
+
+1. Navigate to the `teams management page <https://readthedocs.com/organizations/choose/organization_team_list/>`__.
+2. Click the :guilabel:`<team name>`.
+3. Click :guilabel:`Edit team`
+4. Enable *Auto join users with an organization's email address to this team*.
+5. Click :guilabel:`Save`
+
+With this enabled,
+all users that sign up with SAML will automatically join this team.
+These teams can have either *read-only* or *admin* permissions over a set of projects.
+
+Revoke user's access to all the projects
+----------------------------------------
+
+By disabling access to the SAML integration to a user,
+you revoke access to all the projects the linked Read the Docs user had access to,
+and disable login on Read the Docs completely for that user.
+
+.. warning::
+
+   If the user signed up to Read the Docs previously to enabling SAML on your organization,
+   they may still have access to their account and projects if they were manually added to a team.
+
+   To completely revoke access to a user, remove them from all the teams they are part of.
+
+.. warning::
+
+   If the user was already signed in to Read the Docs when their access was revoked,
+   they may still have access to documentation pages until their session expires.
+   This is three days for the dashboard and documentation pages.
+
+   To completely revoke access to a user, remove them from all the teams they are part of.
```

Comment:
I remember we talked about this a few times, but I'm not sure if we have an issue to track this. I think it's important to find a way of logout the users if they are revoked access. Do we have an issue for this that we can prioritize its research?

Reply:
What we did last time was to decrease the session time, but it's impossible to know if a user access was revoked from the provider (unless we have a webhook of some kind). The other feature we are discussing is having a button to remove the user from all teams with a single click https://github.com/readthedocs/readthedocs-corporate/issues/1476.